### ISSUE TITLE
Plugin support for modifying a Firebase Realtime Database

### ISSUE THREAD

**Issue by ShaneButt** [Arvoria/RobaseService#3]
I do wish to have a plugin made, using Roact, that can modify a Firebase Realtime Database, this would work similarly to the Datastore Editor plugin but, of course, specific to Firebase and designed differently. 

I would aim for this to be as visually pleasing and easy to digest as possible, as well as having it be easily modified, saved, updated, and retrieved.

**Comment by pnlmon** [comment]
I don't think this is needed, Firebase already have a pretty good editor for their database

**Comment by ShaneButt** [comment]
I agree that Firebase have their own editor, my issues are:
- It's not entirely user friendly
- You have to access an external resource (to me this is only a benefit when you're not at your workstation and want to do stuff on your phone) and a tool that can display your database directly in your studio instance is a lot more convenient.


Either way, this wouldn't be a priority feature, but it does fall on the road map.

**Comment by the-scott-davis** [comment]
I agree that an in-studio viewer / editor would be cool. I believe it might be best outside of the scope of the Robase project; perhaps as a separate Studio Plugin?

**Comment by ShaneButt** [comment]
That's the plan! it'd be hosted under its own Repo, this issue is here just to denote the intention to make one in the future (similarly for the `Robase.changed` issue that's closed).
